Dockerfile

```
@@ -42,7 +42,7 @@ RUN tar -cvf valhalla.debug.tar valhalla_*.debug && gzip -9 valhalla.debug.tar
 RUN rm -f valhalla_*.debug
 RUN strip --strip-debug --strip-unneeded valhalla_* || true
 RUN strip /usr/local/lib/libvalhalla.a
-RUN strip /usr/lib/python3/dist-packages/valhalla/python_valhalla.cpython-310-x86_64-linux-gnu.so
+RUN strip /usr/lib/python3/dist-packages/valhalla/python_valhalla.cpython-310-*-gnu.so
```

tbh, not so sure about the `-gnu` either, but should be fine? anyways, better than before for sure.

---

i agree, we can probably just do to be sure of ourselves

```suggestion
RUN strip /usr/lib/python3/dist-packages/valhalla/python_valhalla*.so
```